Formula/thors-sql.rb

```diff
@@ -0,0 +1,58 @@
+class ThorsSql < Formula
+  desc "SQL library for C++14"
+  homepage "https://github.com/Loki-Astari/ThorsSQL"
+  url "https://github.com/Loki-Astari/ThorsSQL.git",
+      :tag => "1.5.6",
+      :revision => "2da4ab817a4f45f0d3f8a7bf7ad0fc455031f0f6"
+
+  depends_on "mysql" => :build
+  depends_on "openssl"
+
+  needs :cxx14
+
+  def install
+    ENV["COV"] = "gcov"
+    system "mysql.server", "start"
```

Comment:
Is this needed to build? Reading https://github.com/Loki-Astari/ThorsSQL/blob/master/doc/building.md makes it seem like that's only for tests.

Reply:
Its run as part of the unit tests.
Which happens during a build.

I could disable this.